### PR TITLE
added snackbar, modal and navigation Components

### PR DESCRIPTION
**Pull request by Balakumar3018**
Ignore resposiveness

### CODE REVIEW

File: Public/components/badge/badge.css [hunk 1]
```diff
@@ -47,13 +47,14 @@
     display: flex;
     justify-content: flex-start;
     align-items: flex-end;
-    font-size: 24px;
+    font-size: 20px;
```
Comment (Koushith) [inline Public/components/badge/badge.css:50]: can make use of relative units here.

**Comment by kr-sarvesh** [comment]
Amazing user interface,
I like the background gray background color you used ,
good to go

File: Public/components/badge/badge.css [hunk 2]
```diff
@@ -65,7 +66,7 @@
     color: var(--primary-color);
     font-size: 16px;
```
Comment (Koushith) [inline Public/components/badge/badge.css:67]: try to use relative units so that it will scale according to the screen size.

**Review by Koushith — APPROVED** [review]
Instead of absolute units go with a relative. rest looks good to me.

**Pull request merged**